lib/simplecov/configuration.rb

```diff
@@ -404,8 +404,21 @@ def enable_coverage(criterion)
       coverage_criteria << criterion
     end
 
+    def primary_coverage(criterion)
+      default_coverage_criterion(criterion)
+    end
+
     def coverage_criteria
-      @coverage_criteria ||= Set[DEFAULT_COVERAGE_CRITERION]
+      @coverage_criteria ||= Set[default_coverage_criterion]
+    end
+
+    def default_coverage_criterion(criterion = nil)
```

Comment:
do we need `default_coverage_criterion`? Can't we just put this into `primary_coverage` and work with that everywhere else? :thinking: